### PR TITLE
CWPSTRUCT, CWPRETSTRUCT, Prop functions and Control Window Style enums

### PR DESCRIPTION
**Pull request by zgabi**
Adding the following new structs, functions and enums:

CWPSTRUCT: https://docs.microsoft.com/en-us/windows/win32/api/winuser/ns-winuser-cwpstruct
CWPRETSTRUCT: https://docs.microsoft.com/en-us/windows/win32/api/winuser/ns-winuser-cwpretstruct

Prop functions:
https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-setpropa
https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getpropa
https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-removepropa

Control Style enums:
https://docs.microsoft.com/en-us/windows/win32/controls/button-styles
https://docs.microsoft.com/en-us/windows/win32/controls/combo-box-styles
https://docs.microsoft.com/en-us/windows/win32/dlgbox/dialog-box-styles
https://docs.microsoft.com/en-us/windows/win32/controls/edit-control-styles
https://docs.microsoft.com/en-us/windows/win32/controls/list-box-styles
https://docs.microsoft.com/en-us/windows/win32/controls/progress-bar-control-styles
https://docs.microsoft.com/en-us/windows/win32/controls/scroll-bar-control-styles
https://docs.microsoft.com/en-us/windows/win32/controls/static-control-styles

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/pinvoke?pullRequest=554) <br/>All CLA requirements met.

File: src/User32/User32+WindowStyles+ComboBoxWindowStyles.cs [hunk 1]
```diff
@@ -0,0 +1,86 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+
+    /// <content>
+    /// Contains the <see cref="ButtonWindowStyles"/> nested type.
```
Comment (AArnott) [inline src/User32/User32+WindowStyles+ComboBoxWindowStyles.cs]: Copy paste error, which I fixed.

**Review by AArnott — APPROVED** [review]
Looks great. Thank you.

**Pull request merged**